change_rf_for.py

```diff
@@ -22,11 +22,22 @@ def next_lines_in_block(lst):
 
     with open(name, mode='r', newline='', encoding='utf8') as fp:
         content = fp.readlines()
+
+        # skip empty files
+        if not content:
+            return False
+
         line_ending = '\r\n' if '\r\n' in content[0] else '\n'    # no special treatment for Mac ;)
-        in_block = False
 
+        in_block = False
+        for_position = None
         added_lines = 0
 
+        # Append newline if file doesn't end with \n
+        if '\n' not in content[len(content) - 1]:
```

Comment:
For the last element it's cleaner just to use `content[-1]`

Reply:
thanks, done ;-)